Mage/src/main/java/mage/game/command/emblems/RadiationEmblem.java

```diff
@@ -0,0 +1,27 @@
+package mage.game.command.emblems;
+
+import mage.abilities.effects.keyword.RadCounterTriggeredAbility;
+import mage.game.command.Emblem;
+
+/**
+ * Special emblem to enable the Rad Counter inherent trigger
+ * with an actual source, to display image on the stack.
+ *
+ * @author Susucr
+ */
+public class RadiationEmblem extends Emblem {
```

Comment:
`Game::init` uses a global effects, why you can't use that code instead fake emblem?

![shot_240408_013044](https://github.com/magefree/mage/assets/8344157/6648167a-b42c-4dc7-b9f4-142cdfa03c73)

Reply:
Because those 3 ability/effect have no source for image. Which is not a problem as they are replacement effects.
But Rad counter has a trigger, which must be shown on the stack.